api.md

```diff
@@ -779,7 +779,237 @@ Get all chunks older than 3 months and newer than 4 months:
 SELECT show_chunks(older_than => interval '3 months', newer_than => interval '4 months');
 ```
 
+## reorder_chunk() [](reorder_chunk)
+Reorder a chunk's heap to follow the order of an index. This function acts
+similarly to the PostgreSQL CLUSTER command (see:
+https://www.postgresql.org/docs/current/sql-cluster.html), however it uses lower
+lock levels so that the chunk and hypertable are able to be read for most of the
+process. It does use a bit more disk space during the operation. This command
+can be particularly useful for data that is accessed when it is older in a
```

Comment:
Start a new paragraph.  Also, how about something instead like:

"This command can be particularly useful when data is often queried in an order different than it was originally inserted.  For example, data is commonly inserted into a hypertable in loose time order (e.g., many devices concurrently sending their current state), but one might typically query the hypertable about a *specific* device.  In such cases, reordering a chunk using an index on `device_id, time` can lead to significant performance improvement for these types of queries."

Reply:
👍